### PR TITLE
Add erebosbase aa1 to master

### LINKED ISSUE #2: There should be a parallax effect in the website
**Is your feature request related to a problem? Please describe.**
Having hard cuts in a website is kinda boring.

**Describe the solution you'd like**
Adding a parallax effect would make the website more dynamic.

### PR DESCRIPTION
**Pull request by petschna**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the website feels kind of bulky, because there is no dynamic in the website #2

**What is the new behavior (if this is a feature change)?**
- Now with the parallax effect the website should feel more dynamic and smooth Closes #2

### CODE REVIEW

**Review by Aichnerc — APPROVED** [review]
LGTM, where's my check?

**Review by petute — APPROVED** [review]
Should be ok to merge.

File: src/Routes.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ import { Route, Switch } from "react-router-dom";
 /**
  * HomePage: A basic template page
```
Comment (kleberbaum) [inline src/Routes.js]:
```suggestion
 * RootPage: A basic template page
```

File: src/components/pages/PageRoot.js [hunk 2]
```diff
@@ -7,17 +7,17 @@ import React from "react";
 import { MDBFreeBird, MDBCol, MDBRow, MDBCardBody, MDBIcon } from "mdbreact";
 
 //> Components
+
```
Comment (kleberbaum) [inline src/components/pages/PageRoot.js:10]:
```suggestion
```

File: src/index.scss [hunk 3]
```diff
@@ -5,9 +5,177 @@
 @import "utilities/base";
 
 // Local
-@import "components/pages/HomePage";
+@import "components/pages/PageRoot";
 
 /** 
  * SPDX-License-Identifier: (EUPL-1.2)
  * Copyright © 2018-2019 Florian Kleber
  */
```
Comment (kleberbaum) [inline src/index.scss]:
```suggestion
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/Routes.js [hunk 4]
```diff
@@ -6,15 +6,15 @@ import { Route, Switch } from "react-router-dom";
 
 //> Components
 /**
- * HomePage: A basic template page
+ * RootPage: A basic template page
```
Comment (kleberbaum) [inline src/Routes.js]:
```suggestion
 * PageRoot: A basic template page
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**